src/app/helpers.js

```diff
@@ -42,10 +43,10 @@ function getStatusStyle(status, property) {
 // Skip team name if `skipTeam` is true.
 // Skip `prefix |` if `prefix` empty.
 function pageTitle(prefix, skipTeam, team) {
-  let suffix = 'Check';
+  let suffix = capitalize(config.appName);
```

Comment:
That's incorrect - either we are not using the function `pageTitle` anymore (which I assume is the case since we replaced it with the `<PageTitle>` component) or this function should use the localizable human string.